src/data/device_registry.ts

```diff
@@ -157,5 +160,23 @@ export const getDeviceIntegrationLookup = (
     }
     deviceIntegrations[entity.device_id!].push(source.domain);
   }
+  // Lookup devices that have no entities
+  if (devices && configEntries) {
+    for (const device of devices) {
+      if (!deviceIntegrations[device.id]) {
+        for (const config_entry_id of device.config_entries) {
+          const entry = configEntries.find(
+            (e) => e.entry_id === config_entry_id
+          );
+          if (entry?.domain) {
+            (
+              deviceIntegrations[device.id] ||
+              (deviceIntegrations[device.id] = [])
+            ).push(entry.domain);
```

Comment:
We will probably add domains multiple time now for devices with entities, should we check if the domain is already in there?

Reply:
The original code above seems to push the domain once per entity already, without ill effect, so I was just going with similar. I could adds checks to avoid duplicates if that's helpful.

Reply:
Maybe best to just make it a set?